packages/babel-generator/test/fixtures/escapes/jsonEscape/output.js

```diff
@@ -1,2 +1,2 @@
 0;// Not a directive
-"\u00A9";
\ No newline at end of file
+"\u00a9";
```

Comment:
You can add an `options.json` to this testcase
```json
{
  "minified": true,
  "jsonCompatibleStrings": true
}	}
```
So it will _not_ be affected by `numbers: true`.